components/dashboard/TextArea/index.tsx

```diff
@@ -124,10 +145,25 @@ const TextArea = ({ shiftRight, setShiftRight }: TextAreaProps) => {
     }
 
     setIsSynced(false);
+  }, [title, input]);
 
-    // DEBOUNCE THE UPDATE FUNCTION
-    localStorage.setItem("editorTitle", title);
-    localStorage.setItem("editorContent", input);
+  useEffect(() => {
+    if (!selectedNoteId) return;
+    if (
+      notes?.find(
```

Comment:
Should put this in a variable, would be good for readability.

Reply:
Yes, I agree.